test/unit/queue/util/createConsumer-test.js

```diff
@@ -193,6 +196,87 @@ test('should acknowledge a message if consumer receives a message', async (t) =>
   t.pass()
 })
 
+test('should receive a message with converted data in onMessage', async (t) => {
+  const { sandbox, connection } = t.context
+
+  const spy = sandbox.spy()
+
+  const consumer = await createConsumer({
+    amqUrl: TEST_AMQ_URL,
+    logger: console,
+    onMessage: sandbox.spy(),
```

Comment:
This should be the `spy` that was created above. This test is not making assertions correctly.